examples/notebooks/solver_selection.ipynb

```diff
@@ -0,0 +1,245 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "expmkveO04pw"
+   },
+   "source": [
+    "## Investigate Different PyBaMM Solvers\n",
+    "\n",
+    "In this notebook, we discuss the process of changing PyBaMM solvers and the corresponding performance trade-offs with each. For further reading on different solvers, see the PyBaMM solver documentation:\n",
+    "\n",
+    "[[1]: PyBaMM Solvers](https://docs.pybamm.org/en/stable/source/api/solvers/index.html#)\n",
+    "\n",
+    "### Setting up the Environment\n",
+    "\n",
+    "Before we begin, we need to ensure that we have all the necessary tools. We will install PyBOP from its development branch and upgrade some dependencies:"
```

Comment:
I would suggest not suggesting to install PyBOP from its development branch (lest you forget changing this at the time of the release!)

Reply:
Thanks for catching this outdated text, it was littered across the notebooks from early pybop releases.